code/modules/admin/verbs/healall.dm

```diff
@@ -1,14 +1,16 @@
 /client/proc/healall()
-    set name = "Heal Everybody"
-    set desc = "Heals every mob in the game"
-    set category = "Fun"
-    if(!src.holder)
-        to_chat(src, "Only administrators may use this command.")
-        return
-    if(!check_rights(R_FUN))
-        to_chat(src, "You need the fun permission to use this command.")
-        return
-    message_admins("[key_name_admin(usr)] healed all mobs")
-    to_chat(world, "<b>The gods have miraculously given everyone new life!</b>")
-    for(var/mob/living/M in world)
-        M.revive(TRUE, TRUE)
\ No newline at end of file
+	set name = "Heal Everybody"
+	set desc = "Heals every mob in the game"
+	set category = "Fun"
+	if(!src.holder)
+		to_chat(src, "Only administrators may use this command.")
+		return
+	if(!check_rights(R_FUN))
+		to_chat(src, "You need the fun permission to use this command.")
+		return
+	if(alert(src, "Confirm Heal All?","Are you sure?","Yes","No") == "No")
+		return
+	message_admins("[key_name_admin(usr)] healed all mobs")
+	to_chat(world, "<b>The gods have miraculously given everyone new life!</b>")
+	for(var/mob/living/M in world)
```

Comment:
This is awful, please loop through one of the mob lists.

Reply:
I mean the original code being in spaces instead of tabs should have been the first red flag 😩